lib/customer.rb

```diff
@@ -20,7 +20,9 @@ def invoices
   end
 
   def transactions
-    customer_transactions = invoices.map { |invoice| invoice.transactions }.flatten
+    customer_transactions = invoices.map do |invoice|
```

Comment:
Useless assignment to variable - `customer_transactions`.<br>Pass `&:transactions` as an argument to `map` instead of a block.